backend/src/main/java/ca/gov/dtsstn/cdcp/api/data/entity/PreferredLanguageEntity.java

```diff
@@ -0,0 +1,66 @@
+package ca.gov.dtsstn.cdcp.api.data.entity;
+
+import java.time.Instant;
+
+import org.immutables.builder.Builder;
+import org.springframework.core.style.ToStringCreator;
+import org.springframework.lang.Nullable;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+
+@Entity(name = "PreferredLanguage")
+@SuppressWarnings({ "serial" })
+public class PreferredLanguageEntity extends AbstractEntity {
```

Comment:
I think this should just be called 'language'

And I think there should be an 'isoCode' (for example, en-CA), and an 'msLocaleId' (or something similar) field for the powerplatform code (ex: 1033).

See <https://learn.microsoft.com/en-us/openspecs/office_standards/ms-oe376/6c085406-a698-4e12-9d4d-c3b0ee3dbc4a>

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/48123208/a6950730-8528-421a-88f4-cee4ceec2ab7)

Reply:
updated